surveys/2024-annual-survey/translations/ja.md

```diff
@@ -326,41 +326,41 @@ Columns:
 
 Type: free form
 
-### Rust の次の側面にどのように優先的に取り組むべきだと思いますか?
-
-Type: matrix
-
-Rows:
-
-- 実行時のパフォーマンス
-- コンパイル時間
-- バイナリサイズ
-- メモリ使用量 (つまり、rustc がコンパイル時に使用する RAM の量)
-- ディスク容量の使用量（ターゲットフォルダーのサイズなど）
-- コンパイラのバグ (ICE、別名内部コンパイラ エラー、コンパイルミスなど)
-- コンパイラのエラーメッセージ
-- IDE エクスペリエンス
-- デバッグ体験
-- ドキュメント (rustdoc、docs.rs)
-- ビルドシステム (cargo)
-- パッケージ管理 (crates.io)
-- 新しい言語機能
-- Rust言語と標準ライブラリのドキュメント
-- コンパイラのバグに遭遇する（例：ICE、つまり内部コンパイラエラーや誤ったコンパイル）
-- 不明瞭なコンパイラエラーメッセージが表示される
-- 標準以下の IDE サポート (例: 一部のエラーが表示されなかったり、分析が遅い)
-- 標準以下のデバッグ経験（例：欠損値の視覚化や非同期スタックトレース）
-- Rust言語や標準ライブラリのドキュメントが不足している
-
-Columns:
-
-- 高優先度
-- 中優先度
-- 低優先度
-
-### 上記以外で優先してほしい機能はありますか?
-
-Type: free form
+### 以下の項目はあなたのRustでのコーディングの生産性にどのように影響を与えていますか？
+
+Type: matrix (optional)
+
+Challenges:
+
+- さまざまなサイズのタプルにロジックを実装すること
+- コードをクレートに分割すること(e.g. orphan ruleによる制約)
+- イテレータを手動で実装する必要があること
+- const fnでできることが限られていること
+- 動的ライブラリプラグインの実装
+- 他の言語との相互運用(e.g. CやC++)
+- 非同期コードで構造化された並行性を達成すること
+- 非同期ランタイムに依存しない非同期コードを書くこと
```

Comment:
[question]
Original is `Writing executor-agnostic async code`.
In this context, I interpret`executor` to mean "async runtime". Is my understanding correct?